README.md

```diff
@@ -733,21 +719,20 @@ more repair work you will have to do later.
 
 *The surface has been repaired;* horizontal fibers are now continuous.  The inline and crossline slices show
 the location of the original (magenta) and repaired (cyan) surfaces.  Note that these overlap in one half
```

Comment:
I assume these colours are the same regardless of the user being in light or dark mode.
Let me know if this is correct.

Reply:
Yes.  The colors (magenta and cyan) were set by the user, and so do not depend on light/dark mode.  But I'm glad you checked.